src/lib/infrastructure/gateway/did-gateway/endpoints/list-did-rules-endpoint.ts

```diff
@@ -0,0 +1,60 @@
+import { DIDRulesDTO, ListDIDRulesDTO } from "@/lib/core/dto/did-dto";
+import { BaseStreamableEndpoint, extractErrorMessage } from "@/lib/sdk/gateway-endpoints";
+import { HTTPRequest } from "@/lib/sdk/http";
+import { Response } from "node-fetch";
+import { convertToDIDRulesDTO, TRucioRules } from "../did-gateway-utils";
+
+export default class ListDIDRulesEndpoint extends BaseStreamableEndpoint<ListDIDRulesDTO, DIDRulesDTO >{
+    constructor(
+        private readonly rucioAuthToken: string,
+        private readonly scope: string,
+        private readonly name: string,
+    ){
+        super(true)
```

Comment:
when do we use `true` as an argument and when do we leave it out?

Reply:
Use it whenever you want data to be pasted as NDJSON. False will tell the parser to assume plain text.